include/zeda/zeda_compat.h

```diff
@@ -76,6 +80,10 @@
  __DEF_WINDLL
  */
 #ifdef __WINDOWS__
+#include <windows.h>
```

Comment:
This line seems unnecessary because windows.h is included above as
```C
/* for Windows */
#if defined(__Windows) || defined(_WIN16) || defined(_WIN32) || defined(__WIN32__)
# ifndef __WINDOWS__
#  define __WINDOWS__ /* unify compiler-dependent macros */
# endif
# ifdef __BORLANDC__ /* Borland C++ */
#  include <vcl.h>
# endif
# ifdef __MSC_VER /* Microsoft Visual C++ */
#  include <windows.h>
#  undef min /* undefine notorious min/max macros */
#  undef max
# endif
#endif
```
If this is really necessary, it means that the case check by referring `__MSC_VER` does not work appropriately. In that case, please suggest a right option.